CONTRIBUTING.md

```diff
@@ -23,6 +23,10 @@ The prerequisites for building, testing, and deploying from this repository are:
 
 * The [.NET SDK](https://get.dot.net/).
 * The [Rust toolset](https://www.rust-lang.org/tools/install)
+* uniffi-bindgen-cs which you can install with the following command: `cargo install uniffi-bindgen-cs --git https://github.com/NordSecurity/uniffi-bindgen-cs --tag v0.7.0+v0.25.0`
+* Make sure git submodule support is enabled with the following command: `git submodule update --init :/`
+* Perl [Windows Download](https://strawberryperl.com/)
+* [Zingolib](https://github.com/nerdcash/zingolib/tree/3d22f26a3d893f751858734d15c78958cf216392) - On Windows you can download [Protobuf](https://github.com/protocolbuffers/protobuf/releases) from here and put it in your path. To install the "Build Tools" on Windows launch "Visual Studio Installer" that you installed for Rust then click "More" → "Import Configuration" and import the ".vsconfig" file at the root of this project and install any components that come up.
```

Comment:
I'm not sure why 'ZingoLib' is listed as a dependency when it isn't installed, and comes in automatically via the git submodule. 
I see you calling out that protobuf needs to be installed, as well as importing .vsconfig. Why the link to ZingoLib (and to a particular commit as well)?

Reply:
Now that you point this out I can see the confusion. The link is to the Zingolib build instructions because the module is pulled in automatically but I still had to follow the 1st time build instructions that the Zingolib folks give, however there build instructions are mostly for Linux so here I mention the windows version of those instructions. Maybe I could update the text to say such. The exact link is just the exact library version that the submodule is currently using. Do you think I should change the link to point to the latest? I'll also check to see if I can deep link down to just the build instructions as I thought I was already doing that.

Reply:
> The link is to the Zingolib build instructions

The link is to a particular commit in the repo, root directory. Not to any build instructions in particular. That's probably what's confusing me.

>  I'll also check to see if I can deep link down to just the build instructions as I thought I was already doing that.

Sounds great.

> The exact link is just the exact library version that the submodule is currently using. Do you think I should change the link to point to the latest? 

I don't want to maintain this doc link to match the submodule each time we update the submodule, as I'll probably forget. So it's probably better to link to the latest, as the submodule will _usually_ be close to the latest.

Reply:
Okay I updated the text as discussed. I believe this is now addressed.